content/pages/disability-benefits/claims-appeal.md

```diff
@@ -23,66 +23,71 @@ When the VBA Regional Office (RO) receives your Notice of Disagreement (NOD), yo
     </p>
   </li>
   <li class="list-group-item">
-    <h4>RO certifies your appeal to the Board</h4>
+    <h4>Your appeal is sent to the Board</h4>
     <p>
-      The RO will prepare your appeal and send it to the Board for review. Your appeal will wait in line until the Board is able to review it.
+    The Decision Review Officer will finish their review and transfer your case to the Board of Veterans’ Appeals. The Board reviews cases in the order they are received as determined by the date of the VA Form 9. A judge will begin work on your appeal when it is among the oldest appeals that are ready for their review.
     </p>
-  </li>
-  <li class="list-group-item">
-    <h4>The Board activates your appeal</h4>
     <p>
-      The Board now has control of your appeal. Lawyers who are experts in Veterans law and reviewing benefit claims will review it. The review prepares your appeal for a Veterans Law Judge who will make a decision. In most cases, your appeal is reviewed in the order it was received. In some cases, an appeal will be moved to the front of the line—this happens if you’re facing severe financial hardship, over age 75, or have a serious illness.
+    If you are suffering a serious illness or are in financial distress, or for other sufficient cause, you can apply to have your appeal Advanced on Docket. If you are older than 75, your appeal will receive this status automatically. Advanced on Docket appeals are prioritized so that they are always at the front of the line. Learn how to request Advanced on Docket status.
     </p>
   </li>
   <li class="list-group-item">
-    <h4>(Optional) Hearing</h4>
+    <h4>You have a hearing (optional visual treatment)</h4>
     <p>
-      If you request a Board hearing, you’ll be sent a letter at least 30 days before it’s scheduled to happen. <a href="/disability-benefits/claims-appeal/hearings/">Learn more about hearings</a>.
+      You have the option to request a hearing with a Veterans Law Judge. At your hearing, the judge will ask you questions about your appeal. A transcript of your hearing will be made and added to your appeal file. The judge will not make a decision about your appeal at the hearing. <a href="/">Learn more about hearings at the Board</a>.
     </p>
   </li>
   <li class="list-group-item">
     <h4>The Board makes its decision</h4>
     <p>
-      The Board has reviewed your appeal and provides a decision on each issue in your appeal. Each issue will be decided in 1 of 3 ways:
+      The Board reviews your appeal and provides a decision on each issue in your appeal. The Board decides each issue in 1 of 3 ways:
```

Comment:
@gnakm should the updated content for this refer to a judge or a board?

Reply:
That's a good question. It should be Board here. 

For more content, you will probably see us using judge and Board interchangeable. At a broader level, we say Board as the location and place where a decision is being made "Board made a decision.." but when something is being specifically reviewed, you will notice the use of judge more.

Reply:
Thanks for the context @gnakm.